lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -57,7 +69,8 @@ class LoadFastEnough4Pwa extends Audit {
     const override = context.settings.throttlingMethod === 'provided' ||
       !isDeepEqual(context.settings.throttling, mobileThrottling);
 
-    const displayValueFinal = override ? displayValueTextWithOverride : displayValueText;
+    const displayValueFinal = override ?
```

Comment:
nit: a bit weird that `displayValueFinal` is not the final display value :)

wdyt about `displayValueTemplate`?